src/index.ts

```diff
@@ -184,17 +188,6 @@ export class Payload {
     if (typeof options.onInit === 'function') options.onInit(this);
   }
 
-  sendEmail = async (message: Message): Promise<unknown> => {
-    const email = await this.email;
-    const result = email.transport.sendMail(message);
-    return result;
-  }
-
-  getMockEmailCredentials = async (): Promise<TestAccount> => {
```

Comment:
@jmikrut Was getMockEmailCredentials used by anything?

Reply:
It was used before to provide the user with a way to retrieve them within their own apps, but now that we are logging that info ourselves within Payload, there is no need for it!